### DISCUSSION TITLE
Can we disable the social link "Share" option

### DISCUSSION THREAD

**Discussion by Pingu-Pox** [requarks/wiki#4066] [Help / Questions]
In the "Social Links" part of each page, there are options to bookmark, share, and print. Can we disable the "share" portion? Can't seem to find anything helpful in the documentation or admin page.

**Comment by lunardigs** [comment]
We'd like to see this as well.
Thank you.

**Comment by gpatkinson** [comment]
Another vote for this feature. I'd like to use Wiki.js for an internal document wiki at my small business. We're not interested in having staff share docs to reddit, facebook, etc or get the impression that that sort of thing would be okay. I tried forking and figuring it and disabling the buttons in the client code, but haven't succeeded as of yet--if anyone else has a working fork, please let me know!

**Comment by gpatkinson** [comment]
For those still looking for a solution, this worked for me:

https://github.com/madodig/wikijs-customization
